.github/workflows/ci.yml

```diff
@@ -196,20 +196,28 @@ jobs:
       - name: Install tools
         run: make init
 
-      - name: Run checks/linters
+      - name: Run required checks/linters
+        env:
+          REVIEWDOG_GITHUB_API_TOKEN: ${{ secrets.GITHUB_TOKEN }}
         run: |
           # use GITHUB_TOKEN because only it has access to GitHub Checks API
-          bin/golangci-lint run -c=.golangci-required.yml --out-format=line-number | env REVIEWDOG_GITHUB_API_TOKEN=${{ secrets.GITHUB_TOKEN }} bin/reviewdog -f=golangci-lint -name='Required checks' -reporter=github-pr-check
-
-          # use ROBOT_TOKEN for better reviewer's name
-          bin/golangci-lint run --out-format=line-number | env REVIEWDOG_GITHUB_API_TOKEN=${{ secrets.ROBOT_TOKEN }} bin/reviewdog -f=golangci-lint -name='Linters' -reporter=github-pr-review
+          bin/golangci-lint run -c=.golangci-required.yml --out-format=line-number | bin/reviewdog -f=golangci-lint -name='Required checks' -reporter=github-pr-check
 
           # run it like that until some of those issues/PRs are resolved:
           # * https://github.com/quasilyte/go-consistent/issues/33
           # * https://github.com/golangci/golangci-lint/issues/288
           # * https://github.com/reviewdog/errorformat/pull/47 (once it is atually used by reviewdog)
           bin/go-consistent -pedantic ./...
 
+      - name: Run optional checks/linters
+        env:
+          REVIEWDOG_GITHUB_API_TOKEN: ${{ secrets.ROBOT_TOKEN }}
+        run: |
+          # use ROBOT_TOKEN for better reviewer's name
+          if [[ -z "${{ secrets.ROBOT_TOKEN }}" ]]; then
```

Comment:
This will run if the secret token is empty, right? I think we want the opposite.